lib/miq_automation_engine/service_models/miq_ae_service_service_template_catalog.rb

```diff
@@ -0,0 +1,7 @@
+module MiqAeMethodService
+  class MiqAeServiceServiceTemplateCatalog < MiqAeServiceModelBase
+    expose :service_template_catalogs, :association => true
```

Comment:
This is not an exposed relationship on the rails side: https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template_catalog.rb#L6-L7

This line just needs to be removed.